main.py

```diff
@@ -794,6 +794,7 @@ def ui_access_wrapper(self, *args, **kwargs):
         admin.NumberOfDeletionRequestsHandler),
     get_redirect_route(r'/frontend_errors', FrontendErrorHandler),
     get_redirect_route(r'/logout', base.LogoutPage),
+    get_redirect_route(r'/sessionBegin', base.SessionBeginHandler),
```

Comment:
Just noting that this doesn't follow the snake_case format of the other handlers -- is the discrepancy actually needed?

Reply:
Nope, fixed.